snakemake_executor_plugin_googlebatch/executor.py

```diff
@@ -361,6 +361,13 @@ def get_allocation_policy(self, job):
             allocation_policy.network = network_policy
         return allocation_policy
 
+        # Add custom compute service account
+        service_account = self.get_param("service_account")
```

Comment:
It looks like you are setting the variable "service_account" but then checking "compute_service_account" - I think that will error.